### PR TITLE
Change default TPC core count for Stargate

### PR DESCRIPTION
**Pull request by tjake**

### CODE REVIEW

**Review by jimdickinson — APPROVED** [review]
👍

**Review by dimas-b — APPROVED** [review]

File: persistence-dse-6.8/src/main/java/io/stargate/db/dse/DsePersistenceActivator.java [hunk 1]
```diff
@@ -112,6 +113,7 @@ public static Config makeConfig(File baseDir) throws IOException {
     c.storage_port = listenPort;
     c.listen_address = listenAddress;
     c.broadcast_address = listenAddress;
+    c.tpc_cores = FBUtilities.getAvailableProcessors();
```
Comment (dimas-b) [inline persistence-dse-6.8/src/main/java/io/stargate/db/dse/DsePersistenceActivator.java:116]: Why not `FBUtilities.getAvailableProcessors() - 1`?

Comment (mpenick) [reply persistence-dse-6.8/src/main/java/io/stargate/db/dse/DsePersistenceActivator.java:116]: Isn't this already done if tpc_cores is not set? Looking for the source code line. I've also tested it by running it in docker with different numbers for `--cpus`.

**Review by mpenick — APPROVED** [review]

File: persistence-dse-6.8/src/main/java/io/stargate/db/dse/DsePersistenceActivator.java (hunk 1 above)
Comment (tjake) [reply persistence-dse-6.8/src/main/java/io/stargate/db/dse/DsePersistenceActivator.java:116]: Since SG is coordinator only it can use all cores.

**Pull request merged**